### PR TITLE
feat(connectors): Add environments

### PR DESCRIPTION
**Pull request by janelletavares**
# Description of change

*A brief description of the change, what it is and why it was made.*

Fixes https://github.com/meroxa/cli/issues/215
Fixes https://github.com/meroxa/cli/issues/216

- [x] Depends on https://github.com/meroxa/meroxa-go/pull/85 (needs to be merged first and then updated to its latest version here).

# Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

# How was this tested?

- [X]  Unit Tests
- [ ]  Tested in staging

# Demo

**Before this pull-request**

```
❯ meroxa connectors ls
  ID               NAME                  TYPE                        STREAMS                      STATE    PIPELINE 
====== ============================= ============= ============================================ ========= ==========
 2479   resource-942792-2630-203946   jdbc-source   (output) resource-2630-240664-public.Users   running   default  

❯ meroxa connectors describe resource-942792-2630-203946                          
       ID:   2479                                       
     Name:   resource-942792-2630-203946                
     Type:   jdbc-source                                
  Streams:   (output) resource-2630-240664-public.Users 
    State:   running                                    
 Pipeline:   default    
```

**After this pull-request**

```
❯ meroxa connectors ls                                  
                 UUID                    ID               NAME                  TYPE                        STREAMS                      STATE    PIPELINE   ENVIRONMENT 
====================================== ====== ============================= ============= ============================================ ========= ========== =============
 24ac3762-3a56-4fc3-8bd4-29647faffe21   2479   resource-942792-2630-203946   jdbc-source   (output) resource-2630-240664-public.Users   running   default    common      

❯ meroxa connectors describe resource-942792-2630-203946
        UUID:   24ac3762-3a56-4fc3-8bd4-29647faffe21       
          ID:   2479                                       
        Name:   resource-942792-2630-203946                
        Type:   jdbc-source                                
     Streams:   (output) resource-2630-240664-public.Users 
       State:   running                                    
    Pipeline:   default                                    
 Environment:   common 
```

### CODE REVIEW

File: utils/display.go [hunk 1]
```diff
@@ -227,6 +227,21 @@ func ConnectorTable(connector *meroxa.Connector) string {
 			{Text: connector.Trace},
 		})
 	}
+	var env string
+	if connector.Environment != nil {
+		if connector.Environment.Name != "" {
+			env = connector.Environment.Name
+		} else {
+			env = connector.Environment.UUID
+		}
```
Comment (dianadoherty) [inline utils/display.go:236]: When the `connector.Environment != nil ` both name and UUID will be present

**Review by raulb — APPROVED** [review]
@janelletavares LGTM. Do you mind including an output demo in the pull-request description of how it looks before and after this change? This is useful to illustrate these changes that end up being referenced from the changelog.

Thank you.

**Review by dianadoherty — APPROVED** [review]

**Comment by janelletavares** [comment]
> @janelletavares LGTM. Do you mind including an output demo in the pull-request description of how it looks before and after this change? This is useful to illustrate these changes that end up being referenced from the changelog.
> 
> Thank you.

I'm having trouble with login. I'll reach out for help tomorrow.

**Comment by raulb** [comment]
- Updated PR description to include demo
- Used upstream version of meroxa-go once this shipped https://github.com/meroxa/meroxa-go/pull/85.
- Updated PR to include connector UUID to eventually hide IDs everywhere and only rely on UUIDs.

**Pull request merged**